tasks/install.yml

```diff
@@ -1,30 +1,19 @@
 ---
 
-- name: Install kong dependencies
+- name: Add BinTray deb repository
   become: yes
-  apt:
-    name: "{{ item }}"
-    state: installed
-  with_items: "{{ kong.dependencies }}"
+  apt_repository:
+    repo: "deb https://dl.bintray.com/mashape/kong-ubuntu-{{ ansible_distribution_release }}-{{ kong.version | regex_replace('(^[0-9]+\\.[0-9]+).*$', '\\1.x') }}
+      {{ ansible_distribution_release }} main"
+    state: present
 
-# This seems to fail often, retry a few times
-- name: Download kong
+  # We still do support Ansible 1.9 hence "command" with --allow-unauthenticated
+  # instead of apt module :()
```

Comment:
I will update the comment. The `allow_unauthenticated` was added in 2.1 so we still need it :(